staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -317,6 +321,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	InPlacePodVerticalScaling: {Default: false, PreRelease: featuregate.Alpha},
 
+	StreamingCollectionEncodingToJSON: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
can you change the default to false? we can selectively test the new feature in prod.

Reply:
updated.